<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TP1_polymorphismeClass</class>
 <widget class="QMainWindow" name="TP1_polymorphismeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TP1_polymorphisme</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_nom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelprenom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_adresse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_titre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Age :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_telephone">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Téléphone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>122</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_professeur">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Professeur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_etudiant">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Etudiant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajout_personne">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajout d'une personne</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_age">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telephone">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>__-__-__-__-__</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>                     @</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionajouter"/>
    <addaction name="actionaffichage"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAffichage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionajouter">
   <property name="text">
    <string>ajouter</string>
   </property>
  </action>
  <action name="actionaffichage">
   <property name="text">
    <string>affichage</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TP1_polymorphisme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton_professeur</sender>
   <signal>toggled(bool)</signal>
   <receiver>TP1_polymorphismeClass</receiver>
   <slot>onRadioButtonProfesseur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_etudiant</sender>
   <signal>toggled(bool)</signal>
   <receiver>TP1_polymorphismeClass</receiver>
   <slot>onRadioButtonEtudiant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRadioButtonProfesseur()</slot>
  <slot>onRadioButtonEtudiant()</slot>
 </slots>
</ui>
